api/providers/ContactProvider.cs

```diff
@@ -46,4 +47,23 @@ public Contact Delete(Guid id)
     {
         throw new NotImplementedException();
     }
+
+    public Contact? GetOrCreateContact(SupplierRequest request)
+    {
+
+        if (request.Contact_id != null)
```

Comment:
```suggestion
        if (request.ContactId!= null)
```

Graag PascalCase aanhouden. (en vranderen in de ContactDTO)